SimG4CMS/Forward/src/ZdcSD.cc

```diff
@@ -197,33 +192,23 @@ double ZdcSD::getEnergyDeposit(const G4Step * aStep) {
       float costh = hit_mom.z()/sqrt(hit_mom.x()*hit_mom.x()+
                                      hit_mom.y()*hit_mom.y()+
                                      hit_mom.z()*hit_mom.z());
-      float th = acos(std::min(std::max(costh,float(-1.)),float(1.)));
+      float th = acos(std::min(std::max(costh,-1.f),1.f));
       // just in case (can do both standard ranges of phi):
       if (th < 0.) th += twopi;
 
       // theta of cone with Cherenkov photons w.r.t.direction of charged part.:
       float costhcher =1./(nMedium*beta);
-      float thcher = acos(std::min(std::max(costhcher,float(-1.)),float(1.)));
+      float thcher = acos(std::min(std::max(costhcher,-1.f),1.f));
 
       // diff thetas of charged part. and quartz direction in LabRF:
-      float DelFibPart = fabs(th - thFibDirRad);
+      float DelFibPart = std::abs(th - thFibDirRad);
 
       // define real distances:
-      float d = fabs(tan(th)-tan(thFibDirRad));   
+      float d = std::abs(tan(th)-tan(thFibDirRad));   
```

Comment:
```std::``` in front of ``` tan``` as well?

Reply:
Will do in the next iteration.

Reply:
sorry, when is this next iteration planned? Not for 10_2_X I assume at this point

Reply:
@fabiocos , I mean Monday. If this PR will be merged I can make other one, if not will do, what can be done during Monday. Main problem of forward SD - there are about 5 classes, which are nearly identical but this was not obvious from the beginning.  Now I know this but there is no time before pre6.